static/js/rendered_markdown.js

```diff
@@ -57,6 +57,14 @@ exports.set_name_in_mention_element = function (element, name) {
     }
 };
 
+// function to make message horizontally scrollable.
+(function () {
+    const prop = document.createElement("style");
+    prop.innerHTML = ".rendered_markdown {" + "overflow-x: auto;" + "}";
+    const ref = document.querySelector("script");
+    ref.parentNode.insertBefore(prop, ref);
+})(window);
```

Comment:
We shouldn't need to add code here; we should be able to just add CSS in static/styles/rendered_markdown.scss for these table elements directly.